src/CounterInterface.php

```diff
@@ -4,19 +4,13 @@
 
 namespace Yiisoft\Yii\RateLimiter;
 
-/**
- * CounterInterface describes rate limiter counter.
- */
 interface CounterInterface
 {
     /**
      * @param string $id Set counter ID.
      * Counters with non-equal IDs are counted separately.
+     *
+     * @return CounterState
      */
-    public function setId(string $id): void;
-
-    /**
-     * Counts one request as done and returns object containing current counter state.
-     */
-    public function incrementAndGetState(): CounterState;
+    public function hit(string $id): CounterState;
```

Comment:
how is this name better?

Reply:
It looks like an attempt to hit an id. I have no idea what does "hit an id" could mean. The previous naming version with an additional description in the comment was more straight-forward. Please, revert this change.

Reply:
I agree that `hit()` may be a bit confusing, but `incrementAndGetState` is more ugly naming, method does't need to explain what it return, just what it does.

Reply:
We should add a description at least